x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -41,256 +34,41 @@ First take a look to the [**Development Best Practices**](#development-best-prac
 Then check check [**Folder structure**](#folder-structure) section to know where is the best place to put your test, [**Test data**](#test-data) section if you need to create any type
 of data for your test, [**Running the tests**](#running-the-tests) to know how to execute the tests and [**Debugging your test**](#debugging-your-test) to debug your test if needed.
 
-Please, before opening a PR with the new test, please make sure that the test fails. If you never see your test fail you don’t know if your test is actually testing the right thing, or testing anything at all.
+Note that we use tags in order to select which tests we want to execute:
+
+```typescript
+export const tag = {
+  SERVERLESS: '@serverless',
+  ESS: '@ess',
+  BROKEN_IN_SERVERLESS: '@brokenInServerless',
+};
+```
```

Comment:
I think once all decisions on the tagging approach are made, this will deserve a more elaborate description and maybe its own section in the README, where we'd explain when to use each tag and give a few examples.

Reply:
Agree.